src/op/declare/collective_comm.cc

```diff
@@ -54,23 +54,12 @@ void Reduce(const CallValues& call) {
   const auto* args = call->args.as<CommReduceArgs>();
   CHECK(args != nullptr);
   ir::Array<Value> ret;
```

Comment:
remove this line

Reply:
the declaration line? I don't understand why

Reply:
```
ir::Array<Value> ret;
```
it's useless now